main.rb

```diff
@@ -0,0 +1,42 @@
+require './app'
+
+class Main
+  def initialize
+    @app = App.new
+  end
+  puts 'Welcome to School Library'
+  def run
+    loop do
+      puts "\n"
+      puts '1 - List all books'
+      puts '2 - List all people'
+      puts '3 - Create a person'
+      puts '4 - Create a book'
+      puts '5 - Create a rental'
+      puts '6 - List all rentals for a given person id'
+      puts '7 - Exit'
+
+      input = gets.chomp
+
+      case input.to_i
+      when 1
+        @app.list_books
+      when 2
+        @app.list_people
+      when 3
+        @app.create_person
+      when 4
+        @app.create_book
+      when 5
+        @app.create_rental
+      when 6
+        @app.list_rentals
+      when 7
+        return
```

Comment:
[OPTIONAL] For a more friendly user experience, I suggest you add a friendly message when a user finish using the app😉.
 ```suggestion
        puts "Thank you for using this app!"
        return
```

Reply:
Thank you for your suggestion. We'll consider adding a friendly message when a user finishes using the app for a better user experience. 😁